postgres/datadog_checks/postgres/postgres.py

```diff
@@ -563,23 +561,21 @@ def _get_service_check_tags(cls, host, tags):
         service_check_tags = list(set(service_check_tags))
         return service_check_tags
 
-    def get_connection(self, host, port, user, password, dbname, ssl, tags, use_cached=True):
+    def _connect(self, host, port, user, password, dbname, ssl, tags):
```

Comment:
Should I make this a @property db method instead?

```
@property
def db():
    if self._db:
            <check status>
    else:
            ....
    return self._db
```

Reply:
IMHO it's better to keep things as it, it's more explicit.